packages/vite/src/buildFeServer.ts

```diff
@@ -1,31 +1,30 @@
-import type { PluginBuild } from 'esbuild'
-import { build as esbuildBuild } from 'esbuild'
 import { build as viteBuild } from 'vite'
 
-import {
-  getRouteHookBabelPlugins,
-  transformWithBabel,
-} from '@redwoodjs/babel-config'
 import { buildWeb } from '@redwoodjs/internal/dist/build/web'
-import { findRouteHooksSrc } from '@redwoodjs/internal/dist/files'
 import { getConfig, getPaths } from '@redwoodjs/project-config'
 
+import { buildRouteHooks } from './buildRouteHooks'
 import { buildRouteManifest } from './buildRouteManifest'
-import { buildRscFeServer } from './buildRscFeServer'
+import { buildRscClientAndWorker } from './buildRscFeServer'
 import { ensureProcessDirWeb } from './utils'
 
 export interface BuildOptions {
   verbose?: boolean
   webDir?: string
 }
 
+// const SKIP = true
+
 export const buildFeServer = async ({ verbose, webDir }: BuildOptions = {}) => {
```

Comment:
Was `buildFeServer` ever the right name for this? We're not actually building a server, are we? We're building *for* (use with) a server. Right?

So maybe `buildForFeServer`? Of if you have any suggestions for better names, just propose something.